core/shared/src/main/scala/monocle/Lens.scala

```diff
@@ -251,6 +251,10 @@ object Lens {
   def codiagonal[S]: Lens[S \/ S, S] =
     PLens.codiagonal
 
+  /** creates a [[Lens]] from a Store comonad coalgebra */
+  def storeFCoalg[F[_], S, A](f: S => F[S])(implicit CS: ComonadStore[F, A]): Lens[S, A] =
```

Comment:
what do you think of replacing ComonadStore by IndexedStore? It would make the relation with PTraversal.applyN more obvious.